### PR TITLE
[DROOLS-4518] Style improvements for CoverageScenario panel

### PR DESCRIPTION
**Pull request by yesamer**
@dupliaka @gitgabrio @danielezonca Can you please test it and review it?

This PR contains some CSS styles change for CoverageScenarioPanel.
The considerable part is the requirement to put the percentage of decision tested INSIDE the hole of the Donut chart.

Related PR: https://github.com/kiegroup/appformer/pull/860

https://issues.jboss.org/browse/DROOLS-4518

Screenshot from 2019-11-28 10-56-31

### CODE REVIEW

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageReportDonutPresenter.java [hunk 1]
```diff
@@ -72,19 +78,38 @@ public void showCoverageReport(final int executed,
                                                 displayer.asWidget());
     }
 
+    /**
+     * Scope of this method is to manage the labels inside the Donut chart. The requirements is to:
```
Comment (gitgabrio) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/main/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageReportDonutPresenter.java]: @yesamer 
Thanks!!!!

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageScenarioListPresenterTest.java [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    @Test
+    public void addScesimDataGroup_RULE() {
+        int scenarioIndex = 1;
+        String scenarioDescription = "description";
+        String expectedLabel = ScenarioSimulationEditorConstants.INSTANCE.rulesFired() + " " + scenarioIndex + ": " + scenarioDescription;
+        commonAddScesimDataGroup(ScenarioSimulationModel.Type.RULE, scenarioIndex, scenarioDescription, expectedLabel);
+    }
+
+    private void commonAddScesimDataGroup(ScenarioSimulationModel.Type type, int scenarioIndex, String scenarioDescription, String expectedLabel) {
+;        Scenario scenario = new Scenario();
```
Comment (jomarko) [inline drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageScenarioListPresenterTest.java]: Please remove the leading `;`

**Comment by jomarko** [comment]
jenkins execute full downstream build

File: drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageScenarioListPresenterTest.java (hunk 2 above)
Comment (yesamer) [reply drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-client/src/test/java/org/drools/workbench/screens/scenariosimulation/client/rightpanel/CoverageScenarioListPresenterTest.java]: @jomarko Done, thank you!

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
build failed but wars generated, @dupliaka please let us know when you download them. afterwards build can be restarted

**Comment by jomarko** [comment]
@yesamer I am sorry I can not approve, I started the war file produced by jenkins but both on firefox anc chrome I do not see the text in the donut whole as you demonstrate in screenshot. See my screenshot.

Screenshot from 2019-12-06 06-34-39

**Comment by yesamer** [comment]
@jomarko Hi Jozef, the changes are on new TestScenario - Coverage Report dock, you are looking the wrong panel. Contact me if you need other info.

**Comment by jomarko** [comment]
Ok, sorry my mistake, I see it now.

**Pull request merged**